Package.swift

```diff
@@ -14,6 +14,9 @@ import PackageDescription
 
 let package = Package(
     name: "SwiftPM",
+    platforms: [
+        .macOS(.v10_10), .iOS(.v9),
```

Comment:
I don't think we need to specify these.

Reply:
Rest of the changes look good! Can you remove this section since we're using availability for newer APIs.

Reply:
Should be removed now